tutorials/R/R-skills/basic-R-skills/R-dplyr-GREPL-Summarise-Piping/R-dplyr-GREPL-Summarise-Piping.Rmd

```diff
@@ -374,6 +374,11 @@ class(myData)
 # View class of 'dataBySpSex' object
 class(dataBySpSex)
 
+```
+```{r compare-classes-2, eval=FALSE}
```

Comment:
please add ,comment=NA as a code chunk option to this line

Reply:
Done